ebuild-writing/common-mistakes/text.xml

```diff
@@ -405,7 +418,8 @@ right page if they want to find out more about the package. And make sure
 the <c>DESCRIPTION</c> is not overly long. Good descriptions will describe
 the main function of the package in a sentence. If no homepage for the package
 is available, set the <c>HOMEPAGE</c> variable to
-<c>https://wiki.gentoo.org/wiki/No_homepage</c>.
+<c>https://wiki.gentoo.org/wiki/No_homepage</c>. Please also strive to test
+HTTPS support for the site used in <c>HOMEPAGE</c>.
```

Comment:
Not your fault, but that paragraph talks about HOMEPAGE first, then about DESCRIPTION, then about HOMEPAGE again. Maybe it should be split in two separate paragraphs, first one for DESCRIPTION, second for HOMEPAGE?

Reply:
Done!